bus-mapping/src/operation.rs

```diff
@@ -555,6 +563,12 @@ impl Op for AccountOp {
     fn into_enum(self) -> OpEnum {
         OpEnum::Account(self)
     }
+
+    fn reverse(&self) -> Self {
+        let mut rev = self.clone();
+        swap(&mut rev.value, &mut rev.value_prev);
+        rev
+    }
```

Comment:
Seeing that you always impl `reverse` on the same way. You can directly impl this function in the trait definition.

On that way, there's only one place with this code and it's always consistent.

Reply:
Currently each struct has their own `value` and `value_prev` type, I can't think a way to make this into trait's common function, would you like to give me a example to follow and improve this?